releasenotes/notes/fix-pulse-parameter-formatter-2ee3fb91efb2794c.yaml

```
@@ -0,0 +1,6 @@
+---
+fixes:
+  - |
+    Fixed a bug in :func:`qiskit.pulse.utils.format_parameter_value` function that
+    unintentionally converts certain integer numbers into float values.
+    See qiskit/#11971 for details.
```

Should this be formatted as a link or does something do that automatically?

```suggestion
    See `qiskit/#11971 <https://github.com/Qiskit/qiskit/issues/11971>`__  for details.
```